src/omotes_sdk/queue_names.py

```diff
@@ -14,22 +15,21 @@ def omotes_exchange_name() -> str:
         return "omotes_exchange"
 
     @staticmethod
-    def job_submission_queue_name(workflow_type: WorkflowType) -> str:
-        """Generate the job submission queue name given the workflow type.
+    def job_submission_queue_name() -> str:
+        """Generate the job submission queue name.
 
-        :param workflow_type: Workflow type.
         :return: The queue name.
         """
-        return f"job_submissions.{workflow_type.workflow_type_name}"
+        return "job_submissions"
 
     @staticmethod
-    def job_results_queue_name(job: Job) -> str:
+    def job_results_queue_name(job_uuid: uuid.UUID) -> str:
         """Generate the job results queue name given the job.
 
-        :param job: The job.
+        :param job_uuid: The identifier of the job.
         :return: The queue name.
         """
-        return f"jobs.{job.id}.result"
+        return f"jobs.{job_uuid}.result"
 
     @staticmethod
     def job_progress_queue_name(job: Job) -> str:
```

Comment:
Is there a reason to have `job_progress_queue_name()` and `job_status_queue_name()` both accept `Job` to generate the queue name, while `job_results_queue_name()` is now accepting `job_uuid`

Reply:
Good idea to make it consistent: https://github.com/Project-OMOTES/omotes-sdk-python/pull/61/commits/c239f062aedb1ad419a44d4cd35fb65e2fa30806